server/controllers/users.js

```diff
@@ -138,20 +162,124 @@ class UserController {
             model: models.Profile,
             as: 'profile',
             attributes: ['firstname', 'lastname', 'bio', 'avatar', 'location']
-          },
-        ],
+          }
+        ]
       });
 
-      return res.status(200)
-        .send({
-          status: 'success',
-          message: 'Users and corresponding profiles retrieved successfully',
-          payload: users
-        });
+      return res.status(200).send({
+        status: 'success',
+        message: 'Users and corresponding profiles retrieved successfully',
+        payload: users
+      });
     } catch (error) {
       next(error);
     }
   }
+
+  /**
+   * Forgot Password
+   * @async
+   * @param  {object} req - Request object
+   * @param {object} res - Response object
+   * @param {object} next
+   * @return {json} Returns json object
+   * @static
+   */
+  static async forgotPassword(req, res, next) {
+    try {
+      const { email } = await validateForgotPassword(req.body);
+
+      const user = await User.findOne({ where: { email } });
+
+      if (!user) return Response.error(res, 404, 'User does not exist');
+
+      // Check if the account is active
+      if (!user.active) return Response.error(res, 401, 'Your account is not verified');
+
+      const token = randomString({ length: 40 });
+      const resetDetails = {
+        resetPasswordToken: token,
+        resetPasswordExpiry: Date.now() + 3600000 // 1 hour from now
```

Comment:
too short (1 day is good), however, it's better this value is not hardcode but gotten from an environment variable (with a default value supplied so things don't break)